library/options.inc.php

```diff
@@ -2861,6 +2883,14 @@ function generate_display_field($frow, $currvalue)
             }
             $i++;
         }
+    } elseif ($data_type == 53) {
+        $service = new EncounterService();
+        if (!empty($currvalue)) {
+            $encounter = $service->getEncounterById($currvalue);
+            if (!empty($encounter)) {
+                $s = $row['date'] ?? '';
+            }
+        }
     }
```

Comment:
is doesn't look like the save part of things has been done yet

Reply:
actually, disregard above comment since bet just using the default save mechanism

Reply:
Yep, using default save.